README.md

```diff
@@ -78,3 +78,4 @@ To make testing simple, the extension provides the `HazelcastServerTestResource`
 ## Limitations (native mode)
 - Default Java serialization is not supported
 - User code deployment is not supported
+- Hazelcast SPI support is limited
```

Comment:
Could you mention how it's limited? Does Kubernetes work? Do custom SPIs work?

Reply:
To be honest, it's hard to define. The code from `parse` method is effectively a normal IMDG code stripped from some _magic_ needed for the sake of OSGi and Glassfish compatibility so I expect some problems in these areas provided that you can make GraalVM work with that.

Naturally, all standard SPI-enabled extensions provided at build time do work properly